api/controllers/ExportController.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
     // auth check
     let ok = false
     try {
-      ok = await GroupMembership.hasModeratorRole(req.session.userId, p.groupId)
+      ok = await GroupMembership.hasModeratorRole(req.session.userId, p.groupId, {}, Responsibility.constants.RESP_ADMINISTRATION)
```

Comment:
lets change the name of this function to hasRole, or really shouldnt it be hasResponsibility?